src/Compiler.php

```diff
@@ -437,13 +437,20 @@ private function preparePropertiesForInclude(array $variables, bool $isRootNode
             } elseif ($name === '__DATA_SCOPED_STYLE_ATTRIBUTE__') {
                 unset($variables[$key]);
                 if ($hasScopedStyleAttribute) {
-                    continue;
+                    foreach ($variables as $variable) {
+                        if ($variable->getName() === 'dataScopedStyleAttribute') {
```

Comment:
A constant would make sense to avoid easy misspelling

Reply:
I maybe might add that later.